site/templates/nav.jet

```diff
@@ -13,7 +13,7 @@
         </form>
       {{end}}
       
-      <button class="navbar-toggler" type="button" s72-collapse=".navbar-collapse" aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
+      <button class="navbar-toggler" type="button" toggle-function="toggleMobileMenu" s72-collapse=".navbar-collapse" aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
Where does this new attribute get used?

Reply:
[Here](https://github.com/shift72/core-template/blob/51d1dd5ba0ad9d508d944b75608d96cc1d3c4451/site/static/js/main.js#L407).